hitl_breakpoint_example.py

```diff
@@ -0,0 +1,186 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
```

Comment:
@mpangrazzi as we discussed, I'd appreciate it if you could take a look at this script (from the point of view of integrate HiTL in Hayhooks)